### PR TITLE
Update name and URL of "OSS-EC NXP MPXA4250A 00000057"

### PR DESCRIPTION
**Pull request by per1234**
This PR consists of two changes:

- Change URL from `https://github.com/RLL-Blue-Dragon/NXP_MPXA4250A_00000057.gi` to `https://github.com/RLL-Blue-Dragon/OSS-EC_NXP_MPXA4250A_00000057.git` (companion to https://github.com/arduino/library-registry/pull/1885)
- Change name from `NXP_MPXA4250_BSL00000057` to `OSS-EC NXP MPXA4250A 00000057` (resolves https://github.com/arduino/library-registry/issues/1893)

Since the name change operation on the database is actually a removal followed by automated re-indexing on the next job run, the URL update will occur as a matter of course. For this reason, the only operation required from the backend maintainer is a standard name change procedure.

### CODE REVIEW

**Comment by per1234** [comment]
I see the request from the library maintainer was closed, probably due to the error reported by the bot:

https://github.com/arduino/library-registry/pull/1885#issuecomment-1251748501

```text
ERROR: Library name OSS-EC NXP MPXA4250A 00000057 not found in the Library Manager index. Library names are not allowed 
       to change after being added to the index.                                                                        
       See: https://github.com/arduino/library-registry/blob/main/FAQ.md#how-can-i-change-a-librarys-name               
       (Rule LP018)
```

I will determine how to proceed and then take the appropriate action for this PR. Until then, I'll leave it open, but marked as "on hold".

**Review by umbynos — APPROVED** [review]

**Comment by umbynos** [comment]
Done in https://github.com/arduino/library-registry/pull/1885#issuecomment-1259455512

**Pull request merged**
